ops/Makefile

```diff
@@ -0,0 +1,26 @@
+# Makefile for az cli shortcuts for SimpleReport operations
+
+SHELL:=/bin/bash # I gave up on being compatible with ye olde "sh"
+SOURCE_SLOT?=staging
+TARGET_SLOT?=production
+DEPLOYED_COMMIT?=$(shell git show --abbrev=7 -s --pretty=%h)
+
+default:
+	@echo "Hello, SimpleReport operator!"
+	@echo "You can use 'make promote-ENV' to promote the staging slot of any environment, if you are logged in to azure"
+
+# Internal target: check if we are currently logged in, so we get a friendly error if not
+.be-logged-in:
+	@if ! az account show >& /dev/null ; then echo "You must be logged in to the az command line"; exit 1; fi
+
+# Internal target: check if the passed-in wildcard is a known environment name. Hard-coding them because let's be real here.
+.valid-env-%:
+	@case $* in test|dev|demo|training|pentest|stg|prod) ;; *) echo "$* is not a valid environment"; exit 1;; esac
+
+promote-%: .be-logged-in .valid-env-%
+	az webapp deployment slot swap -g prime-simple-report-$* -n simple-report-api-$* --slot $(SOURCE_SLOT) --target-slot $(TARGET_SLOT)
```

Comment:
Note that this waits (with a funky animation on output that I would rather not have but haven't figured out how to get rid of) for the swap to be completed before exiting, which means that we should finally actually have a situation where our deploy jobs don't finish until we've successfully, y'know... deployed.